src/plugins/mer/merrunconfigurationaspect.cpp

```diff
@@ -129,7 +134,18 @@ class MerRunConfigWidget : public QWidget
                 MerQmlLiveBenchManager::offerToStartBenchIfNotRunning();
         });
 
-        layout()->addWidget(qmlLiveWidget);
+        qmlLiveLayout->addWidget(qmlLiveWidget);
+        outerLayout->addLayout(qmlLiveLayout);
```

Comment:
Is the extra layout really (still?) needed?

Reply:
Yes

Reply:
Please elaborate. After dropping it I didn't see a difference in the UI.